src/features/routes/routes.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { RoleList, Role } from './helpers.js';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import { CoursesTabs, GroupsTabs, SecretariesTabs, MentorTabs } from '@/screens/index.js';
+import {
+  Counter, ListOfStudents, NotFound,
+  ListOfCourses, AddCourse, ListOfGroups,
+  Auth, UnAssignedList, ListOfSecretaries,
+  ListOfMentors
+} from '../index.js';
+
+
+export const Routes = () => (
+  <>
+    <Switch>
+      <Route exact path="/" render={() => (<h1 className="m-3">Welcome to the WHAT project!</h1>)} />
```

Comment:
Please, remove this file. Pull actual dev branch and insert route to your component in layout.js